test/test_files/log_files/samus_returns/starter_preset.rdvgame

```diff
@@ -64,7 +64,7 @@
                             "Scan Pulse": {
                                 "num_included_in_starting_pickups": 1,
                                 "included_ammo": [
-                                    1000
+                                    150
```

Comment:
I wonder if we should even have scan give anything

Reply:
Changed it to 0 as scan doesn't consume aeion anyways.

Being 100  % accurate will be a pain for future us as you would need to make sure that it can not exceed 2200 etc. blabla